apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingDelegateChanges/StakingDelegateChanges.constants.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { t } from 'i18next';
+
+import { TransactionIdRenderer } from '../../../../2_molecules/TransactionIdRenderer/TransactionIdRenderer';
+import { TxIdWithNotification } from '../../../../2_molecules/TxIdWithNotification/TransactionIdWithNotification';
+import { translations } from '../../../../../locales/i18n';
+import { dateFormat } from '../../../../../utils/helpers';
+import { StakingDelegateChangeItem } from './StakingDelegateChanges.types';
+
+export const COLUMNS_CONFIG = [
+  {
+    id: 'timestamp',
+    title: t(translations.common.tables.columnTitles.timestamp),
+    cellRenderer: (tx: StakingDelegateChangeItem) => dateFormat(tx.timestamp),
+    sortable: true,
+  },
+  {
+    id: 'transactionType',
+    title: t(translations.common.tables.columnTitles.transactionType),
+    cellRenderer: () => t(translations.stakingHistory.delegate),
+  },
+  {
+    id: 'previousDelegate',
+    title: t(translations.stakingHistory.previousDelegate),
+    cellRenderer: (tx: StakingDelegateChangeItem) => (
+      <TxIdWithNotification href="" value={tx.previousDelegate?.id} />
```

Comment:
Why this doesn't link to RSK explorer as usual? 
<img width="1954" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/e9c5e810-8bf8-46c9-a697-8c872c944432">